### ISSUE TITLE
import/order odd error

### ISSUE THREAD

**Issue by devsnek** [benmosher/eslint-plugin-import#1853]
```js
'use strict';

const path = require('path');
const fs = require('fs');
const {
  Value,

  CreateDataProperty,
  DetachArrayBuffer,
  OrdinaryObjectCreate,
  ToString,
  Type,

  Throw,
  AbruptCompletion,

  ManagedRealm,

  inspect,
} = require('..');
```

```
file.js
  4:12  error  There should be no empty line between import groups  import/order
```

In particular it is warning here:
```
1 | 'use strict';
2 |
3 | const path = require('path');
4 | const fs = require('fs');
               ^^^^^^^^^^^^^
5 | const {
6 |   Value,
7 |
8 |   CreateDataProperty,
...
```

**Comment by ljharb** [comment]
hmm, it seems like it might be warning because of the empty lines inside the destructuring pattern. if you delete those 4, does the error go away?

Also, what's your eslint config?

**Comment by devsnek** [comment]
removing the newlines doesn't seem to fix it.

eslint config: https://github.com/engine262/engine262/blob/parser/.eslintrc.js
